Makefile

```diff
@@ -40,4 +38,10 @@ lint:
 	@hlint src test
 	@echo "No linter errors found"
 
+# Docker doesn't always check for new versions
+check-ci:
+	docker pull ghcr.io/fossas/haskell-dev-tools:8.10.4
+	docker build --tag delete-me -f docker/Dockerfile.lint .
+	docker rmi delete-me
+
```

Comment:
@meghfossa, this should help prevent linter/formatter skew, this runs the exact tools that are used in CI, unless you're unlucky enough to be off by a few seconds.

Reply:
This is helpful - thanks for getting this in!